src/settings/adjustments/SettingsAdjustmentsEditor/SettingsAdjustmentsEditor.js

```diff
@@ -36,47 +35,43 @@ class SettingsAdjustmentsEditor extends Component {
     metadata: PropTypes.object,
   };
 
-  getLastMenu() {
-    const { pristine, submitting } = this.props;
-
-    return (
-      <PaneMenu>
-        <Button
-          id="save-adjustment-button"
-          type="submit"
-          marginBottom0
-          disabled={pristine || submitting}
-        >
-          <FormattedMessage id="ui-invoice.save" />
-        </Button>
-      </PaneMenu>
-    );
-  }
-
   render() {
     const {
       handleSubmit,
       close,
       title,
       metadata,
+      pristine,
+      submitting,
     } = this.props;
 
+    const formFooter = (
+      <FormFooter
+        id="save-adjustment-button"
+        label={<FormattedMessage id="ui-invoice.saveAndClose" />}
+        pristine={pristine}
+        submitting={submitting}
+        handleSubmit={handleSubmit}
+        onCancel={close}
+      />
+    );
+
     return (
       <Layer
         contentLabel="Adjustments editor"
         isOpen
       >
         <form
           id="settings-adjustments-form"
-          onSubmit={handleSubmit}
+          style={{ height: '100vh' }}
```

Comment:
actually, it's a hack. IMHO, Layer should be removed here and this hack might not be required

Reply:
it looks like this without `<Layer>`:
![image](https://user-images.githubusercontent.com/43407139/79219443-f45b4500-7e5a-11ea-9b07-fda3a4a68417.png)

Reply:
k, leave as is